src/main/kotlin/gg/skytils/skytilsmod/utils/RenderUtil.kt

```diff
@@ -863,10 +863,11 @@ object RenderUtil {
     }
 
     fun drawAllInList(element: GuiElement, lines: Collection<String>) {
+        val width = lines.maxOf { ScreenRenderer.fontRenderer.getStringWidth(it) }
```

Comment:
Is there a reason why this is used rather than the scaled width? It doesn't seem to take scaling into account.

Reply:
Imma commit an improvement in a second, for some reason
element.width is the correct width, element.scaleWidth is wrong and can't be bothered to find out why.
![image](https://github.com/Skytils/SkytilsMod/assets/74718793/b0b15bf4-8630-42a4-9caf-70590ccbada4)

Reply:
So tldr: element width is used instead of scaled width since it's wrong for some reason